### PR TITLE
Bug 1993999: Update helper texts

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
**Use an existing storage class**
Screenshot from 2021-08-31 16-03-24
Screenshot from 2021-08-31 16-03-39
Screenshot from 2021-08-31 16-03-48
Screenshot from 2021-08-31 16-03-58

**Create a new storage class using local storage devices**
Screenshot from 2021-08-31 16-13-53
Screenshot from 2021-08-31 16-14-15
Screenshot from 2021-08-31 16-14-46

**Connect an external storage platform -- Red Hat Ceph Storage**
Screenshot from 2021-08-31 16-20-37

**Connect an external storage platform -- IBM FlashSystem Storage**
Screenshot from 2021-08-31 16-21-20

**MultiCloud Object Gateway**
Screenshot from 2021-08-31 16-24-48
Screenshot from 2021-08-31 16-25-00

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/body.tsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
           />
         </FormGroup>
         <FormGroup
-          label="Device Type"
+          label="Device type"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/body.tsx:215]: externalize

File: frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx [hunk 2]
```diff
@@ -94,8 +94,8 @@ export const StorageClassFormProvisoners: ExtensionSCProvisionerProp = Object.fr
         },
         thickProvision: {
           name: 'Enable Thick Provisioning',
+          hintText: 'Volumes will allocate the requested capacity upon volume creation',
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx:97]: externalize

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ export const CreateStorageSystemHeader: React.FC<CreateStorageSystemHeaderProps>
         <Text component={TextVariants.h1}>{t('ceph-storage-plugin~Create StorageSystem')}</Text>
         <Text component={TextVariants.small}>
           {t(
-            'ceph-storage-plugin~StorageSystem is an entity of OpenShift Data Foundation. It represents all of the required storage and compute resources.',
+            'ceph-storage-plugin~Create a storage system to represent your OpenShift Data Foundation system and all its required storage and computing resources.',
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]:
```suggestion
            'ceph-storage-plugin~Create a StorageSystem to represent your OpenShift Data Foundation system and all its required storage and computing resources.',
```
Not sure, but will it be StorageSystem ??

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx (hunk 3 above)
Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]: This is what I found in PatternFly > Capitalization doc:

"Capitalize Red Hat feature names when they’re used as proper nouns or when they refer to a capitalized UI term (like a navigation item). **Write them in lowercase when they’re used to describe generic concepts.**"
This text is present in page heading under `Create StorageSystem` and i felt that it is describing what we will be doing. Not sure myself which is correct. WDYT ??

File: frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx (hunk 2 above)
Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx:97]: This is an `Object.freeze` object. How should I externalize it ?? This object is defined in `ocs-storage-class-params.tsx`

Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx:97]: https://github.com/openshift/console/blob/ef3572aba2ac2feccb1bd64d9fa90086127de7ac/frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx#L97

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx (hunk 3 above)
Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]: Makes sense then.

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx [hunk 4]
```diff
@@ -208,7 +223,7 @@ export const BackingStorage: React.FC<BackingStorageProps> = ({
         <Radio
           label={t('ceph-storage-plugin~Connect an external storage platform')}
           description={t(
-            'ceph-storage-plugin~Can be used to connect an external storage platform to OpenShift Data Foundation.',
+            'ceph-storage-plugin~OpenShift Data Foundation will create a dedicated storage class.',
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:226]:
```suggestion
            'ceph-storage-plugin~OpenShift Data Foundation will create a dedicated StorageClass.',
```

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:226]: cc @rebeccaalpert     ^^

Do we need to do make that change here ?

**Review by afreen23 — COMMENTED** [review]
I am surely not very much sure about capitalization correctness.
It would be better if someone from console can review this PR , so we can understand the rules.

Thanks

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx (hunk 3 above)
Comment (rebeccaalpert) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]: Is `StorageSystem` a resource kind?

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx (hunk 4 above)
Comment (rebeccaalpert) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:226]: @spadgett - Do you have feelings on this?

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx (hunk 3 above)
Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]: Yes, it is a resource of `kind: 'StorageSystem'`.

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx (hunk 4 above)
Comment (spadgett) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:229]: I would use StorageClass here to be consistent.

Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:229]: @afreen23 so we want to do it for all three radio-buttons (both button's label and description)

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx (hunk 3 above)
Comment (rebeccaalpert) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/header.tsx:24]: We've used the CamelCase name in admin console in similar contexts (i.e. "Create a new ImageStreamTag by pushing an image...").

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx (hunk 4 above)
Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx:229]: yes @SanjalKatiyar . Lets do for those as well when we have acks from console.

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**